include/random_walks/exponential_hamiltonian_monte_carlo_exact_walk.hpp

```diff
@@ -0,0 +1,293 @@
+// VolEsti (volume computation and sampling library)
+// Copyright (c) 2021 Vissarion Fisikopoulos
+// Copyright (c) 2021 Apostolos Chalkis
+
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef RANDOM_WALKS_EXPONENTIAL_EXACT_HMC_WALK_HPP
+#define RANDOM_WALKS_EXPONENTIAL_EXACT_HMC_WALK_HPP
+
+#define IN_INSIDE_BODY_TOLLERANCE 1e-10
```

Comment:
shouldn't be named `INSIDE_BODY_TOLLERANCE`?

Reply:
Yes, I agree. Done